lib/helpers/getDomNodeAttributes.js

```diff
@@ -0,0 +1,14 @@
+export function getDomNodeAttributes(element,filter) {
```

Comment:
sometimes `element` is used as argument, sometimes `el`. can choose one for consistency. nbd. btw, hi @chanson  :D